packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -0,0 +1,95 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { CreateContainerError } from "@fluidframework/container-utils";
+import { IDocumentStorageService } from "@fluidframework/driver-definitions";
+import { canRetryOnError, DocumentStorageServiceProxy } from "@fluidframework/driver-utils";
+import { ISnapshotTree, IVersion } from "@fluidframework/protocol-definitions";
+import { Container, RetryFor } from "./container";
+import { getRetryDelayFromError } from "./deltaManager";
+
+export class RetriableDocumentStorageService extends DocumentStorageServiceProxy {
+    private static callsWaiting: number = 0;
```

Comment:
statics will not work the way you use them.
There can be 2 independent containers - they should not share any state.

Reply:
Removed callswaiting.